packages/dev/core/src/Lights/LTC/ltcTextureTool.ts

```diff
@@ -11,12 +11,12 @@ import type { Tuple } from "core/types";
  */
 export interface IAreaLightLTCProvider {
     /**
-     * Linearly trasnformed cossine texture for BRDF.
+     * Linearly transformed cosine texture for BRDF.
      */
     ltc1Texture: BaseTexture;
 
     /**
-     * Linearly trasnformed cossine texture for BRDF.
+     * Linearly trasnformed cossine texture Fresnel Approximation.

```

Comment:
nit: same spelling error here
```suggestion
     * Linearly transformed cosine texture Fresnel Approximation.
```